odh-dashboard/base/role.yaml

```diff
@@ -19,3 +19,35 @@ rules:
       - get
       - list
       - watch
+  - apiGroups:
+      - ''
+    verbs:
+      - create
+      - delete
+      - get
+      - patch
+      - update
+      - watch
```

Comment:
Does it need `list` also?

Reply:
We may as we get more ISVs. I've added `list` to the resources we are `get`ing